backend/schema.js

```diff
@@ -5,13 +5,15 @@ export default gql`
     id: ID!
     title: String!
     author: String!
+    pages: String
```

Comment:
Not a GQL expert, but shouldn't this be a number?  E.g. for sorting by number of pages I'd want a numerical sort.